doc.ddoc

```diff
@@ -1,3 +1,4 @@
+ROOT_DIR =
```

Comment:
I was suggesting ROOT_SLASH instead of ROOT_DIR, because it's meaning is clearer, and I suggested to define it as `$(ROOT)/`.